tasks.py

```diff
@@ -9,15 +9,24 @@
 
 @app.task
 def song_links():
-    page = requests.get('http://www.azlyrics.com/t/taylorswift.html',
-                        headers={
-                            'user-agent': 'Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)'})
+    page = requests.get('http://www.azlyrics.com/t/taylorswift.html', headers={'user-agent': 'Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)'})
     # have to mimic google bot here
     parsed_content = BeautifulSoup(page.content, 'html.parser')
     links = parsed_content.find(id='listAlbum').find_all('a', href=True)
-    links_list = [link.get('href') for link in links]
-    actual_links = [
-        urljoin('http://www.azlyrics.com/lyrics/taylorswift.html', link) for
-        link in links_list]
-    return actual_links
+    for link in links:
+        href = link.get('href')
+        title = link.text
+        url = urljoin('http://www.azlyrics.com/lyrics/taylorswift.html', href)
+        song_lyrics.delay(url, title)
 
+@app.task
+def song_lyrics(url, title):
+    page = requests.get(url, headers={'user-agent': 'Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)'})
+    # have to mimic google bot here
+    parsed_content = BeautifulSoup(page.text.replace('<br>', '').encode('utf-8'), 'html.parser')
+    # parent_div = parsed_content.find("div", {"class": "main-page"})
+    lyrics = parsed_content.find('div', {'class': 'ringtone'}).find_next_sibling('div').text
+    saved_lyrics = open("lyrics.txt", "w")
```

Comment:
Writing all of the lyrics to a single file isn't going to play nicely with doing these in parallel. I think we should have an output directory and right a single file per song and save them using the title.